darkmoon/api/v1/metadata/views.py

```diff
@@ -101,6 +101,45 @@ async def list_metadata(
         raise ServerNotFoundException(status_code=504, detail="Server timed out.")
 
 
+@router.get(
+    "/",
+    responses={
+        422: {"Client Error Response": "Unprocessable Content"},
+        504: {"Server Error Response": "Gateway Timeout"},
+    },
+)
+async def list_metadata(
+    collection: AsyncIOMotorCollection = Depends(get_file_metadata_collection),
+    page: int = Query(
+        0,
+        ge=0,
+        le=18446744073709552,
+        description="The page to iterate to.",
+    ),
+    length: int = Query(10, ge=1, le=500),
+) -> list[MetadataEntity]:
+    """Get list of metadata that matches the parameters in the database.
+
+    Parameters:
+        collection (AsyncIOMotorCollection): The database collection to query.
+        page (int): The page number to iterate to.
+        length (int): The number of items per page.
+
+    Returns:
+        List[MetadataEntity]: List of all documents that match parameters in the
+            database.
+
+    Raises:
+        ServerNotFoundException: Endpoint is unable to connect to mongoDB instance
+    """
+    try:
+        data = await collection.find({}).skip(page * length).to_list(length=length)  # type: ignore # noqa
```

Comment:
Please remove the type ignore and the noqa. If you need to resolve these issues, please resolve them in code rather than by ignoring potential errors.